src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/SubaccountSupervisor.kt

```diff
@@ -1487,13 +1483,16 @@ internal class SubaccountSupervisor(
         if (changes.changes.contains(Changes.subaccount)) {
             parseOrdersToMatchPlaceOrdersAndCancelOrders()
             cancelTriggerOrdersWithClosedOrFlippedPositions()
+            reclaimUnutilizedFundsFromChildSubaccounts()
         }
     }
 
+    private var reclaimingChildSubaccountNumbers = mutableSetOf<Int>()
+
     /**
      * @description Loop through all subaccounts to find childSubaccounts that have funds but no open positions or orders. Initiate a transfer to parentSubaccount.
      */
-    private fun reclaimUnutilizedFundsFromChildSubaccounts() {
+    internal fun reclaimUnutilizedFundsFromChildSubaccounts() {
```

Comment:
is this `internal` just for testing?

Reply:
it looks like this could be private actually right?

Reply:
ah yea added a test for it but realized can just let the subaccount change trigger -- will change it back to private